### PR TITLE
ax-363/Feature/scout page UI update

### PR DESCRIPTION
**Pull request by pearlson16**
# Description
Made the ui for the scouts page responsive to both mobile and web. Updated and create the different filter menus and search filter.

Fixes Jira Ticket # https://athletex.atlassian.net/browse/AX-363 [Frontend] Scouts Page (Complete Filter)

## Type of change
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?
I ran the app and made sure that it ran and looked like the mock up

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have assigned 2 reviewers to check my work
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

Screenshot_1646973544

Screenshot_1646973550

Screenshot_1646973553

Screenshot_1646973561

Screenshot_1646973567

### CODE REVIEW

File: lib/pages/DesktopScout.dart [hunk 1]
```diff
@@ -20,6 +20,9 @@ class _DesktopScoutState extends State<DesktopScout> {
   int sportState = 0;
   List<Athlete> nflList = [];
   List<Athlete> nflListFilter = [];
+  String tt = "All Sports";
```
Comment (ketonkss4) [inline lib/pages/DesktopScout.dart:23]: Can you make this string name meaningful?

File: lib/pages/DesktopScout.dart [hunk 2]
```diff
@@ -20,6 +20,9 @@ class _DesktopScoutState extends State<DesktopScout> {
...
+  String tt2 = "Long";
```
Comment (ketonkss4) [inline lib/pages/DesktopScout.dart:24]: This string name is also too vague

File: lib/pages/DesktopScout.dart [hunk 3]
```diff
@@ -47,147 +52,498 @@ class _DesktopScoutState extends State<DesktopScout> {
...
+
+  IndexedStack buildFilterMenu(
+      double sportFilterTxSz, double sportFilterIconSz) {
+    return IndexedStack(
+      index: _widgetIndex,
+      children: [
+        Container(
+          height: 20,
+          child: Row(
+              //mainAxisAlignment: MainAxisAlignment.spaceEvenly,
```
Comment (ketonkss4) [inline lib/pages/DesktopScout.dart:184]: Delete this if not needed

**Review by ketonkss4 — CHANGES_REQUESTED** [review]

**Review by user2745 — APPROVED** [review]

File: lib/pages/DesktopScout.dart (hunk 1 above)
Comment (pearlson16) [reply lib/pages/DesktopScout.dart:23]: done

File: lib/pages/DesktopScout.dart (hunk 2 above)
Comment (pearlson16) [reply lib/pages/DesktopScout.dart:24]: done

File: lib/pages/DesktopScout.dart (hunk 3 above)
Comment (pearlson16) [reply lib/pages/DesktopScout.dart:184]: i do need this

**Review by pearlson16 — COMMENTED** [review]
I completed the changes

**Review by user2745 — APPROVED** [review]

**Review by ketonkss4 — APPROVED** [review]

**Pull request merged**